run-unit-test.sh

```diff
@@ -15,5 +15,6 @@ virtualenv env
 . env/bin/activate
 pip install -e .
 pip install nose mock coverage
+rm .coverage
```

Comment:
I'm sorry, but experience showed that it _was_ cumulative.  I discovered that when Ed asked me why I had reported 53% coverage, while http://calico-jenkins/coverage/ says 38% :-)

I think the text you cited refers to the data set in memory, not in that file.  It is written rather ambiguously, IMO.